mmseg/apis/test.py

```diff
@@ -92,12 +92,13 @@ def single_gpu_test(model,
             if efficient_test:
                 result = [np2tmp(_) for _ in result]
             results.extend(result)
+            batch_size = len(result)
         else:
             if efficient_test:
                 result = np2tmp(result)
             results.append(result)
+            batch_size = 1
 
-        batch_size = data['img'][0].size(0)
```

Comment:
Will `data['img_meta']` do the trick?

Reply:
Thanks for your reply. I am actually not quite sure about the data's shape here. Does `data['img_meta']` looks like `List[List[dict]]`, where the outer is different augmentation and inner is a batch? So I should use `len(data['img_metas'][0])` here?

Reply:
Yes. It is the same as MMDet.

Reply:
Fix with `data['img_metas']`.